### PR TITLE
Make RowProjector's constructor public to consume it in other packages

### PR DESCRIPTION
**Pull request by hakanmemisoglu**
RowProjector will be used in other projects. This PR will allow it to do so.

### CODE REVIEW

File: hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/RowProjector.java [hunk 1]
```diff
@@ -39,7 +39,7 @@
     private final ExpressionEvalContext evalContext;
 
     @SuppressWarnings("unchecked")
-    RowProjector(
+    public RowProjector(
```
Comment (gierlachg) [inline hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/RowProjector.java:42]: You can access it via `SqlProcessors`. It's not enough?

Comment (hakanmemisoglu) [reply hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/RowProjector.java:42]: Unfortunately, not!

Comment (gierlachg) [reply hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/RowProjector.java:42]: Ok, ok...

**Review by gierlachg — APPROVED** [review]

**Review by viliam-durina — APPROVED** [review]
Is this needed for EE?

**Pull request merged**